src/ert/scheduler/lsf_driver.py

```diff
@@ -134,13 +134,37 @@ async def submit(
         self._iens2jobid[iens] = job_id
 
     async def kill(self, iens: int) -> None:
+        if iens not in self._iens2jobid:
+            logger.error(f"LSF kill failed due to missing jobid for realization {iens}")
+            return
+
+        job_id = self._iens2jobid[iens]
+
+        logger.debug(f"Killing realization {iens} with LSF-id {job_id}")
+        process = await asyncio.create_subprocess_exec(
+            self._bkill_cmd,
+            job_id,
+            stdout=asyncio.subprocess.PIPE,
+            stderr=asyncio.subprocess.PIPE,
+        )
+        stdout, stderr = await process.communicate()
+        if process.returncode:
+            logger.error(
+                f"LSF kill failed with returncode {process.returncode} "
+                f"and error message {stderr.decode()}"
+            )
+            return
         try:
-            job_id = self._iens2jobid[iens]
+            stdout_decoded = stdout.decode()
+        except UnicodeDecodeError:
```

Comment:
I thought we agreed to let the python code crash by itself - maybe don't remember it correctly :)

Reply:
We want to crash hard when we have bugs in our code, but if bkill is suddenly misbehaving (giving binary output e.g.), we might not want to crash.

Reply:
This is replicating the behaviour in the lines below where we just log an error if we cannot find the correct string in bkill's output. By this `except` we are just treating erroneous binary output the same way as erroneous string output.